src/transformers/models/xlnet/modeling_tf_xlnet.py

```diff
@@ -569,7 +581,7 @@ def call(
         input_mask=None,
         head_mask=None,
         inputs_embeds=None,
-        use_cache=True,
```

Comment:
this was very out-of-date `use_cache` should not default to `True` as a function input -> it makes `config.use_cache` useless.